src/views/cards/cards.jsx

```diff
@@ -52,7 +52,7 @@ var Cards = injectIntl(React.createClass({
                                     <FormattedMessage id='cards.viewCard' />
                                     {(
                                         formattedLinks['cards.starterLink'] === englishLinks['cards.starterLink'] &&
-                                        locale !== 'en'
+                                        locale.indexOf('en') === -1
```

Comment:
Given how many times it's used, it would be good to refactor this logic out into something like a `localeIsEnglish` helper method.  It also seems more correct to check if `locale.indexOf('en') !== 0` since if it appears elsewhere, it's not English. The most correct might be `locale.split('_')[0] !== 'en'`.